### PR TITLE
updates for MFT + logs from AnalysisQC

### PR DESCRIPTION
**Pull request by chiarazampolli**
@rpezzi , @catalinristea , @noferini

### CODE REVIEW

**Comment by chiarazampolli** [comment]
No impact on FST.

File: DATA/production/configurations/2021/OCT/apass4/setenv_extra.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ export ARGS_EXTRA_PROCESS_o2_fv0_reco_workflow="--fv0-reconstructor"
 #...
 
 # ad-hoc settings for MFT 
-export CONFIG_EXTRA_PROCESS_o2_mft_reco_workflow="$MFT_STROBELGT"
+export CONFIG_EXTRA_PROCESS_o2_mft_reco_workflow="$MFT_STROBELGT;MFTTracking.forceZeroField=false;MFTTracking.FullClusterScan=true;"
```
Comment (rpezzi) [inline DATA/production/configurations/2021/OCT/apass4/setenv_extra.sh:84]: Hi @chiarazampolli , thanks for the update. Actually what we need for pass4 is `MFTTracking.forceZeroField=true`.

**Comment by chiarazampolli** [comment]
Luckily I asked you to check... Sorry, I made the mistake because I opened the files in ML too, to validate, and I must have inverted them. Now fixed, https://github.com/AliceO2Group/O2DPG/pull/369

**Comment by catalinristea** [comment]
Ciao Chiara,

Looks good to me, also.
Thanks!

Yours,
Catalin.

**Pull request merged**